product.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Appliction for Coin</title>
+    <link rel="stylesheet" href="./src/main.css">
+</head>
+<body>
+    <header>
+        <img src="./assets/coins.jpg" alt="Banner of coins.">
+    </header>
+
+    <main>
+        <h1>Tell us about your coin</h1>
```

Comment:
Be consistent about have tags on new lines. 
Ex: 
`
`
<h1> 
      Tell us about your coin
</h1>
`

Reply:
`<h1>` 
   tell us about your coin
`</h1>`